Resources/Prototypes/Entities/Structures/Piping/Atmospherics/miners.yml

```diff
@@ -11,19 +11,18 @@
     - type: Physics
       canCollide: false
     - type: Fixtures
+      fixtures:
+        fix1:
+          shape:
+            !type:PhysShapeCircle
+            radius: .4
+          density: 190
+          mask:
+            - MachineMask
     - type: Transform
       anchored: true
-    - type: Damageable
-      damageContainer: Inorganic
-      damageModifierSet: Metallic
-    - type: Destructible
-      thresholds:
-        - trigger:
-            !type:DamageTrigger
-            damage: 100
-          behaviors:
-            - !type:DoActsBehavior
-              acts: ["Destruction"]
+    - type: Anchorable
```

Comment:
```suggestion
    - type: Anchorable
      flags:
      - Anchorable
```
Make it anchorable only so people cannot easily yeet them out into space.
Also they keep mining when unanchored. It seems like AtmosDeviceUpdateEvent is only raised on anchored devices causing them to never be disabled here. But that can be fixed in another PR as well.

![grafik](https://github.com/user-attachments/assets/904da9c2-2d6c-4cc9-8e51-d03bf4a842d6)

Reply:
Alright I've made the requested change. It can no longer be unanchored. Only ever anchored. As for the "active" state it's not true. When it's not anchored it actually doesn't mine at all, but the examine text never changes for some reason. I attempted to fix, but I'm stupid. 😞 It seemed really simple.